nixos/tests/incus/incus-tests.nix

```diff
@@ -1,500 +1,488 @@
-import ../make-test-python.nix (
-  {
-    pkgs,
-    lib,
-
-    lts ? true,
-
-    allTests ? false,
-
-    appArmor ? false,
-    featureUser ? allTests,
-    initLegacy ? true,
-    initSystemd ? true,
-    instanceContainer ? allTests,
-    instanceVm ? allTests,
-    networkOvs ? allTests,
-    storageLvm ? allTests,
-    storageZfs ? allTests,
-    ...
-  }:
-
-  let
-    releases =
-      init:
-      import ../../release.nix {
-        configuration = {
-          # Building documentation makes the test unnecessarily take a longer time:
-          documentation.enable = lib.mkForce false;
-
-          boot.initrd.systemd.enable = init == "systemd";
-
-          # Arbitrary sysctl modification to ensure containers can update sysctl
-          boot.kernel.sysctl."net.ipv4.ip_forward" = "1";
-        };
-      };
-
-    images = init: {
-      container = {
-        metadata =
-          (releases init).incusContainerMeta.${pkgs.stdenv.hostPlatform.system}
-          + "/tarball/nixos-image-lxc-*-${pkgs.stdenv.hostPlatform.system}.tar.xz";
-
-        rootfs =
-          (releases init).incusContainerImage.${pkgs.stdenv.hostPlatform.system}
-          + "/nixos-lxc-image-${pkgs.stdenv.hostPlatform.system}.squashfs";
-      };
-
-      virtual-machine = {
-        metadata =
-          (releases init).incusVirtualMachineImageMeta.${pkgs.stdenv.hostPlatform.system} + "/*/*.tar.xz";
-        disk = (releases init).incusVirtualMachineImage.${pkgs.stdenv.hostPlatform.system} + "/nixos.qcow2";
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  cfg = config.tests.incus;
+
+  releases =
+    init:
+    import ../../release.nix {
+      configuration = {
+        # Building documentation makes the test unnecessarily take a longer time:
+        documentation.enable = lib.mkForce false;
+
+        boot.initrd.systemd.enable = init == "systemd";
+
+        # Arbitrary sysctl modification to ensure containers can update sysctl
+        boot.kernel.sysctl."net.ipv4.ip_forward" = "1";
```

Comment:
How does that allow containers to modify sysctls? Can you expand on the mechanism?